package.json

```diff
@@ -8,22 +8,14 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
-    "babel-plugin-polyfill-corejs2": "^0.2.0",
```

Comment:
I don't think that any of these dependencies were being used in the project, but @timbeccue correct me if I am wrong on any of these and I'll add them back in.